antenna-documentation/pom.xml

```diff
@@ -20,6 +20,10 @@
 
     <artifactId>antenna-documentation</artifactId>
 
+    <scm>
+        <url>https://github.com/eclipse/antenna</url>
```

Comment:
Should this be the Eclipse project homepage instead?

Ideally, also a `<connection>` tag would be [added](https://maven.apache.org/pom.html#SCM).